### PR TITLE
zh: update zh translate

### PR DESCRIPTION
**Pull request by yuxintong-github**
zh: update index.md of rabc-groups for content_zh OF release-1.2

Please provide a description for what this PR is for.

And to help us figure out who should review this PR, please 
put an X in all the areas that this PR affects.

[ ] Configuration Infrastructure
[x] Docs
[ ] Installation
[ ] Networking
[ ] Performance and Scalability
[ ] Policies and Telemetry
[ ] Security
[ ] Test and Release
[ ] User Experience
[ ] Developer Infrastructure

### CODE REVIEW

**Comment by yuxintong-github** [comment]
sorry for the mistake here https://github.com/istio/istio.io/pull/4625

here is my new PR @SataQiu , please approve it, thanks a lot.

File: content_zh/docs/tasks/security/rbac-groups/index.md [hunk 1]
```diff
@@ -82,6 +82,23 @@ JWT 声明值可以是字符串或字符串列表;两种类型都受支持。
...
+      name: use-mtls-on-sleep
+    spec:
+      host: httpbin.$NS.svc.cluster.local
+      trafficPolicy:
+        tls:
+          mode: ISTIO_MUTUAL
+    EOF
+    {{< /text >}}
+
+
```
Comment (SataQiu) [inline content_zh/docs/tasks/security/rbac-groups/index.md]: 多个空行，需要删除

Comment (yuxintong-github) [reply content_zh/docs/tasks/security/rbac-groups/index.md]: fixed, thanks

**Comment by SataQiu** [comment]
@geeknoid Automatic merge failed! 
```
Branch protection settings are blocking automatic merging See: https://doc.mergify.io/faq.html#mergify-is-unable-to-merge-my-pull-request-due-to-my-branch-protection-settings
```

**Comment by geeknoid** [comment]
release-1.2 is now archived.

**Pull request closed without merging**